### DISCUSSION TITLE
Support for Policy settings?

### DISCUSSION THREAD

**Discussion by kcgthb** [coder/code-server#6505] [Q&A]
Hi!

Does code-server supports Policy settings, as described in https://code.visualstudio.com/docs/getstarted/settings#_settings-precedence
> Policy settings - Set by the system administrator, these values always override other setting values.

We're trying to define a set of fixed settings for a shared installation of code-server, that users couldn't override in their `User/settings.json` configuration file, and it looks like Policy settings would be the way to go. But we can't really figure out how to use them with `code-server`. Does anyone have any pointers?

Thanks!

**Comment by code-asher** [comment]
Good question, I am not sure if the web version of Code supports policy settings.  The only information I seem to be able to find talks about Windows group policy?

https://code.visualstudio.com/docs/setup/enterprise
https://github.com/microsoft/vscode/issues/147756

We are not currently building code-server for Windows so it might be a moot point, although the linked issue has some todos that could support it on other platforms.
